event-sourcing/event-repository/event-repository-liquibase/src/main/resources/liquibase/snapshot-store-db-changesets/001-initial-schema.changelog.xml

```diff
@@ -11,10 +11,10 @@
             <column name="stream_id" type="uuid">
                 <constraints nullable="false"/>
             </column>
-            <column name="sequence_id" type="BIGINT">
+            <column name="version_id" type="BIGINT">
```

Comment:
This seems to be a change of a liquibase script that is already checked into master.  These changes need to be part of a new script.